readme.md

```diff
@@ -301,6 +301,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Web Page Test](https://www.webpagetest.org)** — An [open source](https://github.com/WPO-Foundation/webpagetest) tool for measuring and analyzing the performance of web pages on real devices. Users can choose to produce a Lighthouse report alongside the analysis of WebPageTest results.
 
+* **[is-website-vulnerable](https://github.com/lirantal/is-website-vulnerable)** - An [open source](https://github.com/lirantal/is-website-vulnerable) Node.js CLI tool that helps with finding publicly known security vulnerabilities in a website's frontend JavaScript libraries.
```

Comment:
some optional suggestions for you. no need to double link to the same place and could deverbosify things a bit :)

```suggestion
* **[is-website-vulnerable](https://github.com/lirantal/is-website-vulnerable)** - An open source Node.js CLI tool that finds publicly known security vulnerabilities in a website's frontend JavaScript libraries.
```

Reply:
Looks good, thanks Patrick!